fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1961,6 +1961,9 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, comment = "The max depth that scalar operator optimization can be applied")
     public static int max_scalar_operator_optimize_depth = 256;
 
+    @ConfField(mutable = true, comment = "scalar operator maximum number of flat children.")
+    public static int max_scalar_operator_flat_children = 1000;
```

Comment:
too less

Reply:
What do you suggest? 1000 might have been a lot.